torchvision/prototype/transforms/functional/utils.py

```diff
@@ -0,0 +1,13 @@
+import types
+
+
+def _from_legacy_kernel(legacy_kernel, new_name=None):
```

Comment:
This is basically copying a function and changing its name. This is `jit.script`'able.

Reply:
Could you explain why we need this helper?

For example, apart from ending up with different objects, what's the difference between

```py
center_crop_image = _from_legacy_kernel(_F.center_crop)
```
and
```py
center_crop_image = _F.center_crop
```
?

Reply:
The difference is that in the former `center_crop_image.__name__ == "center_crop_image"`, while in the latter `center_crop_image.__name__ == "center_crop`. 

I've checked, I don't use the `.__name__` attribute for anything anymore, so this is safe to remove.